crates/wash-lib/src/parser/mod.rs

```diff
@@ -133,58 +133,59 @@ impl TryFrom<RawActorConfig> for ActorConfig {
 }
 #[derive(Deserialize, Debug, PartialEq, Eq, Clone, Default)]
 pub struct ProviderConfig {
-    /// The capability ID of the provider.
-    pub capability_id: String,
     /// The vendor name of the provider.
     pub vendor: String,
+    /// Optional WIT world package for the provider
+    pub package: Option<String>,
+    /// The target operating system of the provider archive. Defaults to the current OS.
+    pub os: String,
+    /// The target architecture of the provider archive. Defaults to the current architecture.
+    pub arch: String,
+    /// The Rust target triple to build for. Defaults to the default rust toolchain.
+    pub target: Option<String>,
+    /// Optional override for the provider binary name, required if we cannot infer this from Cargo.toml
+    pub bin_name: Option<String>,
+    /// The directory to store the private signing keys in.
+    pub key_directory: PathBuf,
 }
 
 #[derive(Deserialize, Debug, PartialEq)]
 struct RawProviderConfig {
-    /// The capability ID of the provider.
-    pub capability_id: String,
-    /// The vendor name of the provider. Optional, defaults to 'NoVendor'.
+    /// The vendor name of the provider.
     pub vendor: Option<String>,
+    /// Optional WIT world package for the provider
+    pub package: Option<String>,
```

Comment:
```suggestion
    pub wit_package: Option<String>,
```

Reply:
is this really only the package? not the namespace + package?

Reply:
Yeah it's both, not like we use it anyways but it's just a standin for capability contract ID